.github/workflows/validation.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       matrix:
         python-version: [3.7, 3.8]
-#        python-version: [3.7, 3.8, 3.9]
+        model: [torch, tf, keras, vggish, wav2vec2, torchopenl3, torchcrepe, vqt]
```

Comment:
torch baseline is `naive`, tf baseline is `tf.naive`, and keras baseline is `keras.naive` -- I believe this is why the checks aren't working.

Reply:
If you check the package init -> https://github.com/neuralaudio/hear-baseline/blob/main/hearbaseline/__init__.py -- the torch baseline is available through hearbaseline. I kind of like that the basic naive one is available just through hearbaseline. But feel free to restructure as you wish.

Reply:
If you do restructure and move the torch naive baseline into a different place we'll just have to update some readmes and the validator checks.